cmd/fleetctl/mdm.go

```diff
@@ -306,3 +253,88 @@ fleetctl get host %s
 		},
 	}
 }
+
+// create a mdm command to wipe the device
+func mdmWipeCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "wipe",
+		Usage: "Wipe a host to erase all content on a workstation.",
+		Flags: []cli.Flag{contextFlag(), debugFlag(), &cli.StringFlag{
+			Name:     "host",
+			Usage:    "The host, specified by identifier, that you want to wipe.",
+			Required: true,
+		}},
+		Action: func(c *cli.Context) error {
+			hostIdent := c.String("host")
+
+			client, host, err := hostMdmActionSetup(c, hostIdent, "wipe")
+			if err != nil {
+				return err
+			}
+
+			config, err := client.GetAppConfig()
+			if err != nil {
+				return err
+			}
+
+			// linux hosts need scripts to be enabled in the org settings to wipe.
+			if host.Platform == "linux" && config.ServerSettings.ScriptsDisabled {
+				return errors.New("Can't wipe host because running scripts is disabled in organization settings.")
+			}
+
+			if err := client.MDMWipeHost(host.ID); err != nil {
+				return fmt.Errorf("Failed to wipe host: %w", err)
+			}
+
+			fmt.Fprintf(c.App.Writer, `
+The host will wipe when it comes online.
+
+Copy and run this command to see results:
+
+fleetctl get host %s`, hostIdent)
+
+			return nil
+		},
+	}
+}
+
+// Does some common setup for the host mdm actions such as validating the host,
+// creating the client, getting the desired host, checking permissions, and
+// ensuring MDM is turned on for the host.
+func hostMdmActionSetup(c *cli.Context, hostIdent string, actionType string) (client *service.Client, host *service.HostDetailResponse, err error) {
+	if len(hostIdent) == 0 {
+		return nil, nil, errors.New("No host targeted. Please provide --host.")
+	}
+
+	client, err = clientFromCLI(c)
+	if err != nil {
+		return nil, nil, fmt.Errorf("create client: %w", err)
+	}
+
+	host, err = client.HostByIdentifier(hostIdent)
+	if err != nil {
+		var nfe service.NotFoundErr
+		if errors.As(err, &nfe) {
+			fmt.Println(hostIdent)
+			return nil, nil, errors.New("The host doesn't exist. Please provide a valid host identifier.")
+		}
+
+		var sce kithttp.StatusCoder
+		if errors.As(err, &sce) {
+			if sce.StatusCode() == http.StatusForbidden {
+				return nil, nil, errors.New("Permission denied. You don't have permission to " + actionType + " this host.")
```

Comment:
nit: the idiomatic way to do this is, which is supper annoying

```suggestion
				return nil, nil, fmt.Errorf("Permission denied. You don't have permission to %s this host.", actionType)
```